vss-extension.json~

```diff
@@ -10,11 +10,11 @@
       "Public"
   ],
 =======
-  "version": "0.1.25",
-  "name": "Kubernetes extension",
-  "description": "Kubernetes extension with k8s endpoint. Which enable kubectl command",
-  "publisher": "TsuyoshiUshio",
->>>>>>> parent of 00763c9... add support to download kubectl
+  "version": "0.1.45",
+  "name": "Kubernetes Ostrich extension",
```

Comment:
Ah! :)
BTW, according to the answer of the guy, we the download spec can be like this. What do you think?

1. If the KubeBinary is null or empty, the task will download the binary.
2. if the downloadversion is null, it will donwload latest.

```
curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
```

3. if the downloadversion is filled and KubeBinary is null or empty, download the specific binary. it must be for Linux. 
4. the binally is "chmod". However, the vsts-lib.task's chmod doesn't work, so we need to fire chmod by myself.

https://kubernetes.io/docs/user-guide/prereqs/